core-accounting-fe/src/core-accounting/mockLines.ts

```diff
@@ -7,44 +7,144 @@ declare global {
 Array.prototype.sample = function () {
   return this[Math.floor(Math.random() * this.length)]
 }
+// count.times do {
+//   create line 1
+//   creatre line 2
+//   transfers += [line 1, line 2]
+// }
+// for each pair of credit / debit lines
+//   grab a user & a company
+//   line 1 has user as account / scope and company as partner account / scope
+//   line 2 has company as account / scope and user as partner account / scope
 
-export async function mockLines({count = 15}): Promise<any> {
+//assuming they can both buy and spend from each other
+//what about transfers?
+let code, partnerCode
+if (Math.random() > .5) {
+  code = 'buy_aqd'
+  partnerCode = 'spend_aqd'
+}
+else { //line 2
+  code = 'spend_aqd'
+  partnerCode = 'buy_aqd'
+}
+
+export async function mockTransfers({ count = 8 }): Promise<any> {
```

Comment:
rename to mockLines

Reply:
also add ```transfer_codes = ['buy_tokens', 'spend_tokens']``` arg to mockLines

Reply:
first line of pair gets code = transfer_codes[0] and second line of pair gets code = transfer_codes[1]